source/common/config/delta_subscription_state.cc

```diff
@@ -124,6 +134,19 @@ void DeltaSubscriptionState::handleEstablishmentFailure() {
 
 envoy::api::v2::DeltaDiscoveryRequest DeltaSubscriptionState::getNextRequestAckless() {
   envoy::api::v2::DeltaDiscoveryRequest request;
+  if (!any_request_sent_yet_in_current_stream_) {
+    populateDiscoveryRequest(request);
```

Comment:
Hmm, why the if and else branch is the same?

Reply:
removed the conditional.